### PR TITLE
Added the section of Offliner packager

### PR DESCRIPTION
**Pull request by SeqLaz**
Added the section of the new functionality of Offliner packager

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3790 Added offliner configuration to the project settings form by @suricactus in #524](https://app.clickup.com/t/2192114/QF-3790)

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 1]
```diff
@@ -294,6 +294,42 @@ Now you should see your project and files on [QFieldCloud](https://app.qfield.cl
...
+
+For the old (QGIS) offliner:
+- Two separate layers are created, each representing the filtered datasets:
+  - Layer 1: Filtered with `id % 2 = 1`
+  - Layer 2: Filtered with `id % 2 = 0`
+
+### Advantages:
+
+1. **Efficiency:**
+   - The new offliner configuration optimizes storage by consolidating filtered layers using the same datasource, enhancing efficiency in data management, and for apply distinct  symbology to the same layer.
```
Comment (faebebin) [inline documentation/get-started/tutorials/get-started-qfc.en.md:326]:
```suggestion
   - The new offliner uses less storage by consolidating filtered layers into a single datasource respecting distinct symbology.
```

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 2]
```diff
@@ -294,6 +294,42 @@ Now you should see your project and files on [QFieldCloud](https://app.qfield.cl
...
+
+2. **Compatibility:**
+   - The Packaging Offliner is designed to package data for offline use with QField. For new projects, it is recommended to use the "Optimized Packager" over the deprecated "QGIS Core Offline Editing." The Optimized Packager offers improved performance and enhanced features for offline data management.
```
Comment (faebebin) [inline documentation/get-started/tutorials/get-started-qfc.en.md:329]:
```suggestion


```
this i would discard

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 3]
```diff
@@ -294,6 +294,42 @@ Now you should see your project and files on [QFieldCloud](https://app.qfield.cl
 
 You will receive notifications for events in which you are not the actor. These notifications are specifically for events that are initiated by other members of your organization or collaborators on your projects.
 
+## Enhanced your projects with the Offliner packaging.
+
+The new Packaging Offliner (in "Settings" of your projects) introduces advanced capabilities, allowing the support of multiple layers using the same data source but with distinct filters. This enhancement offers greater flexibility in offline datasources management. Here's an example to illustrate this feature:
```
Comment (faebebin) [inline documentation/get-started/tutorials/get-started-qfc.en.md:299]:
```suggestion
For new projects, it is recommended to use the new "Optimized Packager" over the deprecated "QGIS Core Offline Editing". Set it under "Packaging Offliner" in the "Settings" tab of your project. The "Optimized Packager" supports consolidating filtered layers of same datasource into a single oflline layer, respecting distinct symbology. This also uses less storage. Here's an example to illustrate this feature:
```

File: documentation/get-started/tutorials/get-started-qfc.en.md (hunk 2 above)
Comment (faebebin) [inline documentation/get-started/tutorials/get-started-qfc.en.md:329]:
```suggestion

```
I would drop this

**Review by faebebin — CHANGES_REQUESTED** [review]
@SeqLaz thanks alot for this.

I just made some rewording suggestions.

For correctness please @suricactus please check :)

Many thanks

**Review by suricactus — APPROVED** [review]

File: documentation/get-started/tutorials/get-started-qfc.en.md [hunk 4]
```diff
@@ -294,6 +294,34 @@ Now you should see your project and files on [QFieldCloud](https://app.qfield.cl
 
 You will receive notifications for events in which you are not the actor. These notifications are specifically for events that are initiated by other members of your organization or collaborators on your projects.
 
+## Enhance your project with the "Optimized Packager"
+
+For new projects, it is recommended to use the new "Optimized Packager" over the deprecated "QGIS Core Offline Editing". Set it under "Packaging Offliner" in the "Settings" tab of your project. The "Optimized Packager" supports consolidating filtered layers of same datasource into a single oflline layer, respecting distinct symbology. This also uses less storage. Here's an example to illustrate this feature:
```
Comment (suricactus) [inline documentation/get-started/tutorials/get-started-qfc.en.md:299]:
```suggestion
We recommend to use the new "Optimized Packager" over the deprecated "QGIS Core Offline Editing" for all your projects. Set the packager under "Packaging Offliner" in the "Settings" tab of your project.

The "Optimized Packager" supports consolidating filtered layers of same datasource into a single oflline layer, respecting distinct symbology but also using less storage. Here's an example to illustrate this feature:
```

**Review by faebebin — APPROVED** [review]

**Comment by faebebin** [comment]
many thanks

**Pull request merged**